io-functions-pay-portal/ActivatePayment/__tests__/handler.test.ts

```diff
@@ -1,11 +1,81 @@
 /* tslint:disable: no-any */
 
-import { HttpHandler } from "../handler";
-
-describe("HttpCtrl", () => {
-  it("should return a string when the query parameter is provided", async () => {
-    const httpHandler = HttpHandler();
-    const response = await httpHandler();
-    expect(response.kind).toBe("IResponseSuccessJson");
-  });
+import { Context } from "@azure/functions";
+import { right } from "fp-ts/lib/Either";
+import { PaymentActivationsPostRequest } from "../../generated/pagopa-proxy/PaymentActivationsPostRequest";
+import { ActivatePaymentHandler } from "../handler";
+
+export const context = ({
```

Comment:
Why the `context` is exported?

Reply:
previously its part of external file under `__mock__` folder. i'll remove it. tnx